challenges.tex

```diff
@@ -0,0 +1,530 @@
+ \label{tab:materials}
+\end{table}
+
+\paragraph{Orders}
+Unless specified otherwise, orders that have to be fulfilled in challenges
+are entered through the web shop
+(see the RefBox workshop in \refsec{sec:qualification})
+by any member of the competing team.
+
+In challenges where only one \ac{RS} is present, teams are responsible to
+order products which can be assembled using the available stations only.
+
+\paragraph{Scoring}\label{sec:scoring}
+While the RefBox may assign points during challenges according to the
+regular RCLL rules, those points do not count towards this competition.
+Instead, the point scoring for each challenge is listed in
+\refsec{sec:challenges}.
+
+\subsection{Field Operators}\label{sec:operators}
+In challenges where mockup machines are used, the actual assembly steps have
+to be performed by human supervisors. Whenever a machine is instructed,
+the RefBox operator announces the required interaction. One field operator may
+proceed to enter the field in order to perform the interaction. Afterwards the
+field has to be left immediately.
+The usual rules for replenishing resources (respecting the limited materials
+\refsec{sec:materials}) apply.
+
+\subsection{Available Challenges for the Primary Competition}
+\label{sec:challenges}
+Challenges have different types and variations (difficulty levels).
+The overall score of the competition is calculated by summing up the score
+in the highest difficulty achieved in each of the challenge types.
+The challenge types of the competition are described in
+\refsec{sec:challenge-navigation}-\ref{sec:challenge-markerless}.
+
+The RefBox is used to log the progress and data for each challenge.
+Participating teams are required to send the corresponding game-report dumps to the OC.
+Once the competition is finished, $5$ bonus points are awarded each time a
+team solved a challenge on a difficulty in the shortest amount of time.
+
+\subsubsection{Navigation Challenge}\label{sec:challenge-navigation}
+Basic navigation task with known obstacles.\\
+\textbf{Task:} Drive to 12 randomly generated target zones. A target zone is
+reached, if any robot remains in that zone for at least 5 consecutive seconds.
+The pose of each robot must be set in the beacon message
+for the RefBox to properly register the zones as visited. The robot may move
+within the zone during the 5 second period.\\
+Variations of this challenge depend on the number of available machines
+(see \reftab{tab:challenge-navigation}).
+Multiple robots may be used to simultaneously to reach multiple target zones.
+Partial points may be awarded in case only a subset of target zones were
+reached.
+
+\begin{table}[!htb]
+ \centering
+ \begin{tabular}{l|l|l|l|l}
+  \multirow{2}{*}{Machines}
+  & \multicolumn{4}{c}{Scoring} \\\cline{2-5}
+	& $\geq$ 4 zones  & $\geq$ 8 zones & 12 zones  & max \\\hline\hline
+	 2 & +2 & +4 & +9 & 15 \\
+	 3 & +3 & +6 & +14 & 23 \\
+	 4 & +4 & +8 & +18 & 30 \\
+ \end{tabular}
```

Comment:
I don't really understand the `max` here. Above it is stated that its 12 zones, so what is the difference between 12 and max?

Reply:
ok now I understand, once you reach 4 you get 2 point, then you get 4 points if you reach 8 and then finally 9 points once you visit all 12 zone, thus the sum is 15, took me some while to get that :D

Reply:
i mention that explicitly now, i hope that avoids confusion